ui/tests/unit/utils/stream-logger-test.js

```diff
@@ -13,7 +13,7 @@ module('Unit | Util | StreamLogger', function() {
     });
 
     logger.start();
-    logger.stop();
+    await logger.stop();
```

Comment:
Oops. How long has this latent race condition been around?

Reply:
indeed 😳